.cycloid.yml

```diff
@@ -37,6 +38,7 @@ config:
         path: 'pipeline/pipeline-split-jobs.yml'
       variables:
         path: 'pipeline/variables.sample.yml'
+        destination: '($ project $)/pipeline/variables-($ environment $).yml'
```

Comment:
@sdurrheimer would you mind checking this one ? is it a problem to have the same destination file for variables for both scenario ? 
what if the user create a pp for each scenario in the same env ?

Reply:
> what if the user create a pp for each scenario in the same env ?

How would do that ? For me, one environment can be associated to one pipeline variant.

Reply:
you're right, I was thinking about using the same environment for difference use-case but you can't have two environments with the same name :wink: